vividus-tests/src/main/resources/story/system/UltrafastGridSteps.story

```diff
@@ -51,11 +51,11 @@ When I run visual test with Applitools UFG using:
 |chrome_mobile_emulation|       |            |Galaxy S10|portrait         |       |
 
 
-Scenario: Validation of step: 'When I run visual test with Applitools UFG using:$applitoolsConfigurations and matrix:$matrix' for context element with element cut
+Scenario: Validation of step: 'When I run visual test with Applitools UFG using:$applitoolsConfigurations and matrix:$matrix' for context element with element cut and Accessibility check
 When I change context to element located by `xpath(.//body)`
 When I run visual test with Applitools UFG using:
-|baselineName            |batchName  |action  |elementsToIgnore|
-|ufg-context-element-cut |<batchName>|<action>|tagName(img)    |
+|baselineName            |batchName  |action  |elementsToIgnore|accessibilityStandard|
+|ufg-context-element-cut |<batchName>|<action>|tagName(img)    |WCAG 2.1 - AA        |
```

Comment:
how can we be sure a11y is verified actually?

Reply:
added known issue for failed check